daemon.go

```diff
@@ -151,29 +151,37 @@ func failColor(format string, args ...interface{}) string {
 func build() bool {
 	log.Println(okColor("Running build command!"))
 
-	args := strings.Split(*flagBuild, " ")
-	if len(args) == 0 {
-		// If the user has specified and empty then we are done.
-		return true
-	}
+	commands := strings.Split(*flagBuild, "&&")
+	success := true
+	for _, c := range commands {
+		c = strings.TrimSpace(c)
+		args := strings.Split(c, " ")
+		if len(args) == 0 {
+			// If the user has specified and empty then we are done.
```

Comment:
This comment does not make sense

Reply:
Yes, that comment make no sense, but this was from another contributor.